utils/jest/preprocessor.js

```diff
@@ -0,0 +1,22 @@
+const mergeAll = require('ramda').mergeAll;
+const babel = require('../../config/babel')();
+const babelJest = require('babel-jest');
+const buildConfigs = require('../buildConfigs');
+const config = require('../kytConfig')();
+
+const { clientConfig } = buildConfigs(config);
+
+// Merge userland babel config with our babel config
+// TODO: This logic should probably move to config/babel.js,
+// but that would be a breaking change.
+const clientBabelConfig = clientConfig.module.loaders
+  .find(loader => loader.loader === 'babel-loader')
+  .query;
+
+// TODO: What's sort of confusing here is that userland babel config
+// (at least for Project-Vi) does not put things in babel.env's.
```

Comment:
Yea for sure! We actually already have an issue to address this #134 so I'll delete this comment. Good catch!